modules/ardrivo/test/String.cxx

```diff
@@ -0,0 +1,237 @@
+#include <catch2/catch.hpp>
+#include "WString.h"
+
+TEST_CASE("Test compare a string to another" "[compareTo]")
+{
+	String test = "test";
+	String test2 = "test";
+	String test3 = "test";
+	String test4 = "est";	
+	REQUIRE(test.compareTo(test2) == false);
+	REQUIRE(test3.compareTo(test4) == true);
+}
+
+TEST_CASE("Tests whether or not a String starts with the characters of another String" "[startsWith]")
+{
+	String test = "test";
+	String test2 = "test";
+	String test3 = "est";
+	String test4 = "test";	
+
+	REQUIRE(test.startsWith(test2) == true);
+	REQUIRE(test3.startsWith(test4) == false);
+}
+
+TEST_CASE("Tests whether or not a String ends with the characters of another String" "[endsWith]")
+{
+	String test = "test";
+	String test2 = "test";
+	String test3 = "est";
+	String test4 = "test";
+
+	REQUIRE(test.startsWith(test2) == true);
+	REQUIRE(test3.startsWith(test4) == false);
+}
+
+TEST_CASE("Copies the String�s characters to the supplied buffer" "[getBytes]")
+{
+	using Catch::Contains; 
```

Comment:
Please dont do 'using ..some namespace..' when you only use it once

Reply:
Well it is not used only once? Or what do you mean?

Reply:
This isn't a `using namespace`; this is equivalent to  `using Contains = Catch::Contains;`

Reply:
Oh my bad didnt read it right but still think its better to remove the `using` and just do `Catch::Contains("test")` the single time its used